_dashboards/selfhost-maps-server.md

```diff
@@ -0,0 +1,98 @@
+---
+layout: default
+title: Self-host maps server
+nav_order: 6
+---
+
+# Self-host maps server
+
+The self-host maps server for OpenSearch Dashboards allows users to access the default maps service on air-gapped environments. The server provides a map manifest url with map tiles and vectors, a map tiles url, and a map vectors url that are compatible with OpenSearch.
+
+The following steps walk through setting up and using the self-host maps server with OpenSearch Dashboards.
+
+You can access the `maps-server` image via the OpenSearch official [Docker Hub repository](https://hub.docker.com/u/opensearchproject).
+{: .note}
+
+## Pull Docker image
+
+Open Docker and then open your terminal to run the following command:
+
+`docker pull opensearch/opensearch-maps-server`
+
+## Set up the server
+
+You must set up the map tiles before running the server. You have two setup options: (1) use the OpenSearch-provided maps service tiles set or (2) generate the raster tile set yourself.
+
+### Option 1: Use the OpenSearch-provided maps service tiles set
+
+Create a Docker volume to hold the tiles set:
+
+`docker volume create tiles-data`
+
+Download the tiles set from the OpenSearch maps service (planet tiles sets [0-8](https://maps.opensearch.org/offline/planet-osm-default-z0-z8.tar.gz) and [0-10](https://maps.opensearch.org/offline/planet-osm-default-z0-z10.tar.gz) are available):
```

Comment:
IMO, we can explicitly show the 0-8 and 0-10 url on doc so that user can be easier to use it in command. Thanks!
https://maps.opensearch.org/offline/planet-osm-default-z0-z8.tar.gz
https://maps.opensearch.org/offline/planet-osm-default-z0-z10.tar.gz